components/need_tag.js

```diff
@@ -1,36 +1,31 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { withStyles } from "@material-ui/core/styles";
+import { globalTheme } from "../theme";
+import { css } from "react-emotion";
 
-const styles = theme => ({
-  needsTag: {
-    marginRight: theme.spacing.unit,
-    marginBottom: theme.spacing.unit,
-    color: "black",
-    borderRadius: 1,
-    display: "inline-flex",
-    padding: "4px 8px"
```

Comment:
missing some ;'s on the end of some of these lines. Noticed that the class is not being applied and think that could be the problem. Also worth removing  the " around inline-flex and 4px 8px

Reply:
good call the quotes was the culprit